networkdb/networkdb.go

```diff
@@ -200,6 +200,17 @@ func (nDB *NetworkDB) Close() {
 	}
 }
 
+// Peers returns the gossip peers for a given network.
+func (nDB *NetworkDB) Peers(nid string) []string {
+	nDB.RLock()
+	defer nDB.RUnlock()
+	peers := make([]string, 0, len(nDB.networkNodes[nid]))
+	for _, nodeName := range nDB.networkNodes[nid] {
+		peers = append(peers, nDB.nodes[nodeName].Addr.String())
```

Comment:
I think it would be useful to return not only the node address but also the node name.
Also it would be more future proof to retr na slice of `NodeInfo` structures, so that we can add new data in future w/o changing the API.